### PR TITLE
Added links to problems

### PR DESCRIPTION
**Pull request by williamfiset**

### CODE REVIEW

File: Sieve.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ## See also
-* [Sieve of Eratosthenes]()
+* [Sieve of Eratosthenes](https://en.wikipedia.org/wiki/Sieve_of_Eratosthenes)
```
Comment (SuprDewd) [inline Sieve.md]: An empty link, such as `[Sieve of Eratosthenes]()`, links to the internal page called "Sieve of Eratosthenes". I think this link should remain internal, and instead the wiki link should go under "External links" on the internal "Sieve of Eratosthenes" page.

File: Topological sorting.md [hunk 2]
```diff
@@ -3,4 +3,5 @@ categories: Graph theory
 ...
 
 ## Problems
-- [Reactivity Series](https://open.kattis.com/problems/reactivity)
\ No newline at end of file
+* [Reactivity Series](https://open.kattis.com/problems/reactivity)
+* [Build deps](https://open.kattis.com/problems/builddeps)
```
Comment (SuprDewd) [inline Topological sorting.md]: The title should be "Build Dependencies". Also, this is a very nice introductory problem, so maybe it should go above Reactivity Series?

Comment (williamfiset) [reply Topological sorting.md]: I did not know you ordered them in terms of difficulty. Good to know.

Comment (SuprDewd) [reply Topological sorting.md]: I try to order them in a way that is useful for someone new to the topic. So some kind of order based on difficulty, simplicity and usefulness.

**Pull request merged**